GitUI/UserControls/RevisionGrid/Columns/MessageColumnProvider.cs

```diff
@@ -287,13 +285,9 @@ void DrawSuperProjectRef(string label, ref int currentOffset)
 
             var arrowType = gitRef.IsSelected
                 ? RefArrowType.Filled
-                : gitRef.IsSelectedHeadMergeSource
-                    ? RefArrowType.NotFilled
-                    : RefArrowType.None;
+                : RefArrowType.None;
```

Comment:
Why should `gitRef.IsSelectedHeadMergeSource` be ignored?

Reply:
I think this is redundant.

![image](https://user-images.githubusercontent.com/3169265/109434131-66f5f480-7a1c-11eb-99ba-678cb7c260a4.png)

Reply:
This shows that this is the current tracking branch, so it is valuable info

Reply:
When does it matter? As I understand it, all branches look at the same commit.

Reply:
One can have multiple remote branches at the same commit. This arrow marks the push target.

Reply:
How to understand this if a commit is in all repositories?

Reply:
This shows the branch you will push to by default

Reply:
![image](https://user-images.githubusercontent.com/3169265/110026944-04f9ff80-7d3a-11eb-9adb-12f6fa65fcd1.png)

1 case: One remote branch - no need to show arrow. (it is not informative)
2 case: Many remote branches - no need to show arrow. (it is not informative because you still need to make a choice from the list, double-check the choice)

Reply:
It is not really the arrow that is important, it is to show that this is the current remote tracked branch.
You do not know if there is a a remote at all for the "test" branch.
This is regardless if there is one or many remotes, if the remote tracked branch is on the same branch as the local or not.

Of course, you can always push to any remote url (with any branch name at the remote) also if they are not tracked, but then you make an active choice.

The "jigsaw" screenshot shown in another thread is a nicer way to show the relation is a better way but not available now..

Reply:
>  you still need to make a choice from the list, double-check the choice

What is your use-case?

I see where it will go when I click `Push` and hit `Enter`. There is no need to chose. The remote-tracking branch could have a different name (not in the screenshots).

![grafik](https://user-images.githubusercontent.com/36601201/110035503-0f1cfd80-7d3c-11eb-8730-e2b0ac6dfd1b.png)

![grafik](https://user-images.githubusercontent.com/36601201/110035808-6753ff80-7d3c-11eb-9900-83cc3e06e599.png)

Reply:
![image](https://user-images.githubusercontent.com/3169265/110082505-18d34f00-7d96-11eb-97d2-b518417e049e.png)

In my experience, I always check where the push will go.
Perhaps you do too, but you don't notice.
If everything is standard, then there is no need to concentrate, and if not, then why do it twice. We may not notice some actions, but they affect.

This is a matter of habit and there is no point in discussing it or I do not see some use case?

Reply:
The point is it's visible when looking at the history. I don't have to hit push just to get the info.  Knowing where the current branch would go to if pushed helps. Why are you so stuck on getting rid of it? Does it bother you so much?  Seeing where my current branch is and where it would push to, I can visually go oh I need to rebase because it's not a fast forward.

Reply:
Do you look in history where the changes you just made will go?